source/pwmout_api.c

```diff
@@ -174,7 +174,7 @@ void pwmout_period_us(pwmout_t* obj, int us)
     SystemCoreClockUpdate();
 
     TimHandle.Init.Period        = us - 1;
-    TimHandle.Init.Prescaler     = (uint16_t)(SystemCoreClock / 1000000) - 1; // 1 �s tick
+    TimHandle.Init.Prescaler     = (uint16_t)(SystemCoreClock / 1000000) - 1; // 1 �s tick
```

Comment:
(pedantic mode: ON) this weird character here was a `µ`; probably a `u` is better